src/index.html

```diff
@@ -1,15 +1,172 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="POST"
+    onsubmit="onSubmit()">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="surname">
+          Surname:
+          <input
+          type="text"
+          id="surname"
+          name="surname"
+          required
+          inlength="3"
+          >
+        </label>
+        <label for="name">
+          Name:
+          <input
+          type="text"
+          id="name"
+          name="name"
+          required
+          maxlength="20"
+          >
+        </label>
+        <label for="age">
+          How old are You?
+          <input
+          type="number"
+          id="age"
+          name="age"
+          value="12"
+          required
+          min="0"
+          max="100"
+          >
```

Comment:
fix an indentation everywhere
```suggestion
          <input
            type="number"
            id="age"
            name="age"
            value="12"
            required
            min="0"
            max="100"
          >
```